.travis.yml

```diff
@@ -43,22 +41,40 @@ script:
 branches:
   only:
     - master
-env:  
-  - TEST_COMMAND='mx eclipseformat --primary' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx checkstyle' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx pylint' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-travis1' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-travis2' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-travis3' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-travis4' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-travis-sulong' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-travis-jruby' LIBXML_LIB_HOME=/usr/lib/x86_64-linux-gnu OPENSSL_LIB_HOME=/usr/lib/x86_64-linux-gnu CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-travis-argon2' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-clangformatcheck' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx checkoverlap' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx canonicalizeprojects' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-httpcheck' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-mdlcheck' CLANG='suite version (3.2)'
-  - TEST_COMMAND='mx su-gitlogcheck' CLANG='suite version (3.2)'
+matrix:
+  fast_finish: true
+  include:
+  - env: TEST_COMMAND='mx eclipseformat --primary' CLANG='suite version (3.2)'
+    jdk: oraclejdk8
```

Comment:
It seems strange to have the `oraclejdk8` for every gate task, but @pointhi found out that Travis fails otherwise: https://travis-ci.org/graalvm/sulong/builds/160955483.